internal/cli/apps.go

```diff
@@ -117,10 +124,60 @@ func appsCmd(cli *cli) *cobra.Command {
 	cmd.AddCommand(showAppCmd(cli))
 	cmd.AddCommand(updateAppCmd(cli))
 	cmd.AddCommand(deleteAppCmd(cli))
+	cmd.AddCommand(useAppCmd(cli))
 
 	return cmd
 }
 
+func useAppCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID   string
+		None bool
+	}
+
+	cmd := &cobra.Command{
+		Use:   "use",
+		Short: "Choose a default application",
+		Long: `auth0 apps use <client-id>
+Specify your preferred application for interaction with the auth0 CLI
+`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if inputs.None {
+				inputs.ID = ""
+			} else {
+				if len(args) == 0 {
+					err := appID.Pick(cmd, &inputs.ID, cli.appPickerOptions)
+					if err != nil {
+						return err
+					}
+				} else {
+					inputs.ID = args[0]
+				}
+			}
+
+			if err := cli.setDefaultAppID(inputs.ID); err != nil {
+				return err
+			}
+
+			if inputs.ID == "" {
+				cli.renderer.Infof("Successfully removed the default application")
+			} else {
+				cli.renderer.Infof("Successfully set the default application to %s", ansi.Faint(inputs.ID))
+				cli.renderer.Infof("%s: You might wanna try `auth0 quickstarts download`", ansi.Faint("Hint:"))
```

Comment:
Can you please add the client id to the hint?

Reply:
Also the hint is showing two colons after `Hint`.

Reply:
Done! Good eye, thanks 💪